src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Exit this page
```

Comment:
From @davidc-gds:

> A very small Q on pattern naming conventions: I think we usually go with ‘a’ instead of ‘the’?
> So, ‘Exit a page quickly’
> Would match ‘Start using a service’ and others.